packages/playground/src/dashboard/components/user_farms.vue

```diff
@@ -206,15 +202,15 @@ export default {
 
     async function getUserFarms() {
       try {
-        const { data } = await gridProxyClient.farms.list({
+        const { data, count } = await gridProxyClient.farms.list({
```

Comment:
the count is not returned by default for the latest release of gridproxy, can you check that?